src/pages/home/base/mobile-core/lifecycle/metrics.md

```diff
@@ -1,112 +0,0 @@
----
```

Comment:
doc changes look good to me, with one request.
can we preserve the original metrics.md file names so we don't break the links?
unless @calebk1 suggests we can rename them and have redirects in place

Reply:
I went ahead and reverted the file rename to preserve external links to the `metrics.md` files.